app/views/errors.py

```diff
@@ -31,7 +31,7 @@ def add_cache_control(response):
 @errors_blueprint.app_errorhandler(NoTokenException)
 def unauthorized(error=None):
     log_exception(error)
-    return _render_error_page(401)
+    return render_theme_template('default', 'session-expired.html'), 401
```

Comment:
If you can find a way to distinguish the 2 in flask them I'm open to feedback. However the only reason someone would hit the 401 would be if they were up to no good, so showing them the Session expired page is probably ok.